### PR TITLE
PR 1570 - Feature/at 9022 step 8 add appropriation funds page

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**
Please get latest SNOW database, as two columns were added that are necessary for a successful testing of this PR.

### CODE REVIEW

File: src/steps/10-FinancialDetails/AppropriationOfFunds.vue [hunk 1]
```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-container class="container-max-width mb-7" fluid>
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">Let’s find out more about your appropriation of funds</h1>
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/AppropriationOfFunds.vue:5]: remove class `page-header` - that's used when no intro paragraph. Add class `mb-3` to the `h1`

File: src/steps/10-FinancialDetails/AppropriationOfFunds.vue [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        <div class="copy-max-width">
+          <p>
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/AppropriationOfFunds.vue:7]: add class `mb-10` to the `p`

File: src/steps/10-FinancialDetails/AppropriationOfFunds.vue [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  },
+})
+
+export default class AppropriationOfFunds extends Mixins(SaveOnLeave) {
+  private fundingRequest: FundingRequestDTO ={};
+  private selectedFundType: "" | "O_M" | "RDT_E" | "PROCUREMENT" = "";
+  private fundTypes: RadioButton[] = [
+    {
+      id: "O&M",
+      label: "Operations & Maintenance",
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/AppropriationOfFunds.vue:62]: Add abbreviations in parentheses to the first 2 labels as in the figma

Screen Shot 2023-05-08 at 3 57 51 PM

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/steps/10-FinancialDetails/AppropriationOfFunds.vue (hunk 1 above)
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/AppropriationOfFunds.vue:5]: done

File: src/steps/10-FinancialDetails/AppropriationOfFunds.vue (hunk 2 above)
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/AppropriationOfFunds.vue:7]: done

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**